recipes/chatgpt/webview.js

```diff
@@ -1 +1,9 @@
-module.exports = Ferdium => Ferdium;
+module.exports = Ferdium => {
+  Ferdium.handleDarkMode(isEnabled => {
+    if (isEnabled) {
```

Comment:
Please collapse this `if` block into a ternary condition since its a single word difference - easier to maintain

Reply:
agree!